umh.docs.umh.app/content/en/docs/development/contribute/documentation/setup-environment.md

```diff
@@ -26,13 +26,12 @@ using [Chocolatey](https://chocolatey.org/)).
 
 - [Git](https://git-scm.com/)
 - [Git LFS](https://git-lfs.com/)
-- [Hugo (Extended version)](https://gohugo.io/)
-- [Node.js LTS](https://nodejs.org/en/)
+- [Hugo (Extended version)](https://gohugo.io/) v0.111.3
```

Comment:
should not be part of this PR

Reply:
Like I said, it's a small change to reflect the new environment setup methods. Since there can be other people that want to contribute to the documentation, I thought that pushing this asap would be a good idea